src/main/java/it/gov/pagopa/pu/workflow/controller/DebtPositionFineControllerImpl.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.pu.workflow.controller;
+
+import it.gov.pagopa.pu.workflow.controller.generated.DebtPositionFineApi;
+import it.gov.pagopa.pu.workflow.dto.FineReductionExpirationRequestDTO;
+import it.gov.pagopa.pu.workflow.dto.generated.WorkflowCreatedDTO;
+import it.gov.pagopa.pu.workflow.service.debtposition.custom.fine.DebtPositionFineService;
+import it.gov.pagopa.pu.workflow.utilities.SecurityUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Slf4j
+public class DebtPositionFineControllerImpl implements DebtPositionFineApi {
+
+  private final DebtPositionFineService debtPositionFineService;
+
+  public DebtPositionFineControllerImpl(DebtPositionFineService debtPositionFineService) {
+    this.debtPositionFineService = debtPositionFineService;
+  }
+
+  @Override
+  public ResponseEntity<WorkflowCreatedDTO> handleFineReductionExpiration(Long debtPositionId, FineReductionExpirationRequestDTO body, Boolean massive) {
```

Comment:
This WF could be manually executed just with the debtPositionId:
* paymentEventRequest should not be used (null when requested by the generic WF)
* FineWfExecutionConfig should be retrieved from the DB reading from debtPositionWorkflowType
* massive should no be used (false when requested by the generic WF)
```suggestion
  public ResponseEntity<WorkflowCreatedDTO> handleFineReductionExpiration(Long debtPositionId) {
```